support-config/src/main/resources/touchpoint.PROD.conf

```diff
@@ -1,10 +1,14 @@
 touchpoint.backend.environments {
   PROD {
     stripe {
-      AUD {
+      AU {
         api.key.public = "pk_live_57xV50eFjPIA990PEGWJwoHp"
         api.key.secret = "dummy-key"
       }
+      US {
+        api.key.public = "pk_live_2O6zPMHXNs2AGea4bAmq5R7Z"
+        api.key.secret = "dummy-key"
```

Comment:
I'm not sure what the dummy-key is for (although it was a useful error id when it went wrong)
Are we having a retro as I do think we could have some it:test for the config which could catch this if they are run before merging.  Same idea as here:
https://github.com/guardian/support-service-lambdas/blob/master/lib/s3ConfigValidator/src/test/scala/com/gu/test/S3ConfigFilesEffectsTest.scala